test_unstructured_ingest/expected-structured-output/Sharepoint/Shared Documents/fake-text.json

```diff
@@ -0,0 +1,110 @@
+[
+  {
+    "type": "NarrativeText",
+    "element_id": "1df8eeb8be847c3a1a7411e3be3e0396",
+    "metadata": {
+      "data_source": {
+        "record_locator": {
+          "site": "https://unstructuredio.sharepoint.com/",
+          "UniqueId": "880f80ca-cebf-48d0-b639-aeb671b3c431",
+          "ServerRelativeUrl": "/Shared Documents/fake-text.txt"
+        },
+        "date_created": "2023-06-16T05:04:55Z",
+        "date_modified": "2023-06-16T05:04:55Z"
+      },
+      "filename": "fake-text.txt",
+      "filetype": "text/plain"
+    },
+    "text": "This is a test document to use for unit tests."
+  },
+  {
+    "type": "Address",
+    "element_id": "a9d4657034aa3fdb5177f1325e912362",
+    "metadata": {
+      "data_source": {
+        "record_locator": {
+          "site": "https://unstructuredio.sharepoint.com/",
+          "UniqueId": "880f80ca-cebf-48d0-b639-aeb671b3c431",
+          "ServerRelativeUrl": "/Shared Documents/fake-text.txt"
+        },
+        "date_created": "2023-06-16T05:04:55Z",
+        "date_modified": "2023-06-16T05:04:55Z"
+      },
+      "filename": "fake-text.txt",
+      "filetype": "text/plain"
+    },
+    "text": "Doylestown, PA 18901"
+  },
+  {
+    "type": "Title",
+    "element_id": "9c218520320f238595f1fde74bdd137d",
+    "metadata": {
+      "data_source": {
+        "record_locator": {
+          "site": "https://unstructuredio.sharepoint.com/",
+          "UniqueId": "880f80ca-cebf-48d0-b639-aeb671b3c431",
+          "ServerRelativeUrl": "/Shared Documents/fake-text.txt"
+        },
+        "date_created": "2023-06-16T05:04:55Z",
+        "date_modified": "2023-06-16T05:04:55Z"
+      },
+      "filename": "fake-text.txt",
+      "filetype": "text/plain"
+    },
+    "text": "Important points:"
+  },
+  {
+    "type": "ListItem",
+    "element_id": "39a3ae572581d0f1fe7511fd7b3aa414",
+    "metadata": {
+      "data_source": {
+        "record_locator": {
+          "site": "https://unstructuredio.sharepoint.com/",
+          "UniqueId": "880f80ca-cebf-48d0-b639-aeb671b3c431",
+          "ServerRelativeUrl": "/Shared Documents/fake-text.txt"
```

Comment:
lets make these as snake case per convention. so: `unique_id` and `server_relative_url`

Reply:
looks like you addressed this.